src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/deciders/QueueHealthDecider.java

```diff
@@ -81,8 +81,8 @@ public Decision operate() {
   }
 
   private void configureActionPriority() {
-    // TODO: Input from user configured yml
-    this.actionsByUserPriority.add(ModifyQueueCapacityAction.NAME);
+    ArrayList<String> actionPriority = getDeciderActionPriorityOrder(this.getClass().getName());
```

Comment:
Use `NAME` instead of `this.getClass().getName()`

Reply:
Updated